pennylane/_device.py

```diff
@@ -205,7 +205,10 @@ def execute(self, queue, expectation):
         with self.execution_context():
             self.pre_apply()
             for operation in queue:
-                self.apply(operation.name, operation.wires, operation.parameters)
+                apply_inverse = False
+                if operation.manual_inverse == 0:
+                    apply_inverse = True
+                self.apply(operation.name, operation.wires, operation.parameters, apply_inverse)
```

Comment:
Doesn't `.manual_inverse` already return a boolean? If that is the case, it is probably more transparent to pass that on to `apply()` directly, without the additional `if` construction.

Reply:
Yes, good catch! This must have been during the end stages of the flight... 😫